### PR TITLE
Build the CI/CD webinar takeover

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Build the CI/CD webinar takeover and the engage page.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Refresh the homepage until you see the new takeover
- Make sure it matches [the design](https://github.com/ubuntudesign/web-squad/issues/994#issuecomment-469625896)
- Make sure both the takeover and engage page matches [the copy doc](https://docs.google.com/document/d/1YfVwlrv41pSai0G0cukHt5ASkshpKOAPK_sZeS7ZmSk/edit#)

## Issue / Card
Fixes https://github.com/ubuntudesign/web-squad/issues/996

### CODE REVIEW

**Comment by anasereijo** [comment]
- [ ] The animation looks nice but can we repeat the animation twice and then stop?

- [ ] Fix the suru shape. On my screen isn't displaying right.

**Comment by anasereijo** [comment]
@anthonydillon ana-mation looks good 👍 thanks for the hard work
Can you check the gradient? Looks a bit different from my design... The left side should be darker. 

thank you

**Comment by anthonydillon** [comment]
@anasereijo updated

**Comment by anasereijo** [comment]
@anthonydillon gracias!

File: templates/takeovers/_ci-cd-webinar.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<section lang="en" class="js-takeover is-bordered p-takeover p-strip--image is-deep is-dark">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h1>Learn DevOps best practices</h1>
+      <p class="p-heading--four u-sv2">
+        Automate your development pipeline with a CI/CD workflow.
+      </p>
+      <p class="u-hide--small">
+        <a itemprop="url" href="/engage/ci-cd-webinar?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_DC_Kubernetes_WBN_CICD" class="p-button--positive">Watch the webinar</a>
```
Comment (caldav) [inline templates/takeovers/_ci-cd-webinar.html]:
```suggestion
        <a itemprop="url" href="/engage/devops-cicd-webinar?utm_source=takeover&utm_campaign=CY19_DC_Kubernetes_WBN_CICD" class="p-button--positive">Watch the webinar</a>
```

File: templates/takeovers/_ci-cd-webinar.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+          c27.9,0,55.6,5.6,81.4,16.5c25,10.5,47.6,25.8,66.7,44.9c39.3,39.2,61.3,92.5,61.3,148c0.1,27.9-5.4,55.5-16.2,81.3
+          c-9.4,22.3-22.6,42.8-39,60.5l-11.6-52.9v-0.1c-0.7-2.3-2.8-4-5.2-4H356c-0.2,0-0.3,0-0.5,0c-1.4,0.3-2.6,1.2-3.4,2.4s-1,2.6-0.7,4
+          l13.6,62.3c0,0.1,0.1,0.2,0.1,0.3c0,0.1,0.1,0.2,0.1,0.3c0.1,0.2,0.1,0.3,0.2,0.5c0,0.1,0.1,0.2,0.1,0.3c0.1,0.2,0.2,0.3,0.3,0.5
+          c0,0,0.1,0.1,0.1,0.1c0.2,0.2,0.4,0.4,0.6,0.6c0.1,0.1,0.1,0.1,0.2,0.2c0.2,0.2,0.4,0.3,0.6,0.5c0.1,0,0.1,0.1,0.2,0.2
+          c0,0,0.1,0.1,0.1,0.1c0.1,0.1,0.2,0.1,0.4,0.2c0.1,0.1,0.3,0.1,0.5,0.2c0.1,0,0.2,0.1,0.3,0.1l58,17.7c0.5,0.2,1.1,0.3,1.6,0.3
+          c0.8,0,1.7-0.2,2.5-0.7c2.6-1.3,3.6-4.5,2.2-7.1C432.3,386.5,431.2,385.5,429.8,385.1z"/>
+        </g>
+      </svg>
+      <p class="u-hide--large u-hide--medium u-no-margin--bottom" style="margin-top: 1rem; max-width: none;">
+        <a itemprop="url" href="/engage/ci-cd-webinar?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_DC_Kubernetes_WBN_CICD" class="p-button--positive">Watch the webinar</a>
```
Comment (caldav) [inline templates/takeovers/_ci-cd-webinar.html]:
```suggestion
        <a itemprop="url" href="/engage/devops-cicd-webinar?utm_source=takeover&utm_campaign=CY19_DC_Kubernetes_WBN_CICD" class="p-button--positive">Watch the webinar</a>
```

File: templates/engage/ci-cd-webinar.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{% extends "engage/base_engage.html" %}
+
+{% block meta_description %}Getting Started Easily With DevOps Practices: CI/CD{% endblock %}
```
Comment (caldav) [inline templates/engage/ci-cd-webinar.html]:
```suggestion
{% block meta_description %}A webinar to learn how to automate Continuous Integration, Continuous Delivery, and Continuous Deployment with Kubernetes{% endblock %}
```

File: templates/engage/devops-cicd-webinar.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+{% block title %}Getting Started Easily With DevOps Practices: CI/CD{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1zAv3ouUrzQcQDw2DSefaePXcMu35BAumIMpzqmIkycs/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--image is-deep js-takeover is-dark p-takeover">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>Getting Started Easily With DevOps Practices: CI/CD</h1>
```
Comment (pmahnke) [inline templates/engage/devops-cicd-webinar.html]:
```suggestion
      <h1>Get started with DevOps best practices: CI/CD</h1>
```

Please change, the other is not sentence case and hard to read

**Pull request merged**